android/res/xml/backup_content_v31.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<data-extraction-rules>
+  <cloud-backup>
+    <include domain="file" path="bookmarks/"/>
+    <include domain="file" path="settings.ini"/>
+  </cloud-backup>
+  <device-transfer>
+    <include domain="file" path="bookmarks/"/>
+    <include domain="file" path="settings.ini"/>
+  </device-transfer>
```

Comment:
> This separation allows you, for example, to exclude a file or directory from Google Drive backups while still transferring it during device-to-device (D2D) transfers. This may be useful if you have files which would be too large to back up to the cloud, but which can be transferred between devices without issue.

I assume D2D transfers are for cases when a user switches to another device, e.g. an upgrade?
In this case we might consider adding map files there, otherwise the user will have to re-download all the maps **manually** and it could be a hassle.

Reply:
> Device-to-device (D2D) transfers: User data is sent directly to the user's new device from their older device, such as by using a cable.

Reply:
> I assume D2D transfers are for cases when a user switches to another device, e.g. an upgrade?
In this case we might consider adding map files there, otherwise the user will have to re-download all the maps manually and it could be a hassle.

I thought about it. Do you think that it worth it?

Reply:
https://support.google.com/android/answer/6193424?hl=en#zippy=%2Cswitch-from-an-android-phone
> 3. When asked, choose to copy apps and data from your old phone.
> 4. If you have a cable to connect your phones, follow the on-screen instructions to copy your data.

I guess that should work with usb-c 2 usb-c cable.

I think we should use this feature for maps and other downloaded files (incl. world/coasts and fonts etc.)! I guess transfer everything except cache.
It will help people with limited bandwidth and will save time re-selecting maps to re-download. It'll also ease map servers load a bit :)

Reply:
Generally, maps should be compatible. This case is unlikely, usually, people transfer everything to an (almost) empty device.
Let's transfer maps too for D2D. Worlds should be transferred only if this feature works also on de-googled devices.

Reply:
=> https://github.com/organicmaps/organicmaps/issues/2060

I don't have spare cycles to test this feature. I don't even know how to use this D2D. I guess that this mode is only used during initial device configuration. I can play with it, but some time later in the future.